app/.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  extends: ["airbnb-typescript", "next", "prettier"],
```

Comment:
add `airbnb` too?

Reply:
I ended up removing airbnb (not their typescript though). Some of their rules were intense, and throwing errors in the out of the box next code. I'll try adding it again if / when we start working on a frontend for this and give them a proper read though.